avm/src/vm/event.rs

```diff
@@ -192,6 +195,80 @@ impl HandlerFragment {
     };
   }
 
+  #[inline(always)]
```

Comment:
what does this do?

Reply:
it ensures that the function gets inlined - originally, we were handling each instruction within `HandlerFragment::run` and I wanted the performance to otherwise be the same since I was pulling the handling into other functions